Dockerfile

```
@@ -0,0 +1,22 @@
+# Builder
+FROM node:alpine as builder
+
+RUN apk add --no-cache git
+
+WORKDIR /src
+
+COPY package.json /src/package.json
+RUN npm install
+
+COPY . /src
+RUN npm run build
+
+
+# App
+FROM nginx:latest
+
+COPY --from=builder /src/webapp /app
```

Why not directly copying the files to the right location instead of going through a symlink ?

```suggestion
RUN rm -rf /usr/share/nginx/html
COPY --from=builder /src/webapp /usr/share/nginx/html
```

---

tbh I think we should keep it as `/app` so that the documentation for how to link your own config is easier to maintain and read.